From 327ccf086e026dca4a3f2c4f078201fac3390128 Mon Sep 17 00:00:00 2001
From: Robert Kanter <rkanter@cloudera.com>
Date: Thu, 10 Jan 2013 22:06:01 -0800
Subject: [PATCH 192/795] MR1: CLOUDERA-BUILD. Simple fix for org.apache.hadoop.streaming.TestMultipleCachefiles.testMultipleCachefiles failing

This code change is already in upstream branch-1
CDH-9766
(cherry picked from commit 61933e12733631e6ac66ba8068d8c7333331cde1)

(cherry picked from commit 20b30b4c57320cda03d219f999bcdf6fdcb64d37)
---
 .../hadoop/streaming/TestMultipleCachefiles.java   |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/contrib/streaming/src/test/org/apache/hadoop/streaming/TestMultipleCachefiles.java b/src/contrib/streaming/src/test/org/apache/hadoop/streaming/TestMultipleCachefiles.java
index 53e32cb..3568d6e 100644
--- a/src/contrib/streaming/src/test/org/apache/hadoop/streaming/TestMultipleCachefiles.java
+++ b/src/contrib/streaming/src/test/org/apache/hadoop/streaming/TestMultipleCachefiles.java
@@ -106,10 +106,10 @@ public void testMultipleCachefiles() throws Exception
       file.writeBytes(mapString2 + "\n");
       file.close();
       file = fileSys.create(new Path(CACHE_FILE));
-      file.writeBytes(cacheString);
+      file.writeBytes(cacheString + "\n");
       file.close();
       file = fileSys.create(new Path(CACHE_FILE_2));
-      file.writeBytes(cacheString2);
+      file.writeBytes(cacheString2 + "\n");
       file.close();
         
       job = new StreamJob(argv, mayExit);     
-- 
1.7.0.4

